backend/src/openarchiefbeheer/urls.py

```diff
@@ -73,5 +73,15 @@
 # For this to work, you need to have built the frontend (`npm run build` in the frontend/ dir)
 if settings.E2E_SERVE_FRONTEND:
     urlpatterns = [
-        path("", TemplateView.as_view(template_name="index.html"), name="frontend")
+        path("", TemplateView.as_view(template_name="index.html"), name="frontend"),
+        path(
+            "login",
+            TemplateView.as_view(template_name="index.html"),
+            name="frontend-login",
+        ),
+        path(
+            "logout",
+            TemplateView.as_view(template_name="index.html"),
+            name="frontend-logout",
+        ),
```

Comment:
```suggestion
        path(
            "login/",
            TemplateView.as_view(template_name="index.html"),
            name="frontend-login",
        ),
        path(
            "logout/",
            TemplateView.as_view(template_name="index.html"),
            name="frontend-logout",
        ),
```
For consistency with the other paths, but could you check that it still works? :sweat_smile: 
Also: could you add a comment about the fact that this was needed because for some yet unknown reason this route was being handled by the backend and not the frontend?

Reply:
- [ ] Test ad document the added paths